postman/schemas/transfer.yml

```diff
@@ -2889,6 +3140,38 @@ components:
           type: string
         amount:
           "$ref": "#/components/schemas/money"
+    adjustmentQuotes:
+      required:
+      - id
+      - label
+      - type
+      - value
+      type: object
+      properties:
+        id:
+          type: string
+          enum:
+          - TransferFee
+          - TransferFeeDiscount
+        label:
+          type: string
```

Comment:
Reminder here to add descriptions so it's clear what these fields all represent.